js/local/local.js

```diff
@@ -169,6 +170,80 @@ wam.initConditionRecommendationTypes = function() {
 			}
 		});
 	}
+  
+    //Live
+    if ($('#P526OccurrenceEditorForm_attribute_318Item_new_0').length) {
```

Comment:
I would recommend moving these identifier selectors into a configuration object, which could just be at the top of this script:

```
var selectors = {
    occurrenceEditorForm: {
        fieldName: '#SomeReallyLong_andStupid_lookingSelector_1_2_3' //, etc
    }
};
```

Then this line would become something like `if ($(selectors.occurrenceEditorForm.whateverFieldThisIsFor).length)`.  There are a few other selectors below as well (`wamcon`) so they could probably go into config as well.

The advantage of this is that you put all the selectors in one place, easy to update, and also are decoupling the implementation of the script from the particular DOM elements you happen to be manipulating.